src/controllers/newuser.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (req, res) => {
+  console.log(req.body);
```

Comment:
we need them!!!!!! to see what's going on
they shall be removed upon completion